neo/renderer/tr_main.cpp

```diff
@@ -473,9 +474,9 @@ void R_AxisToModelMatrix( const idMat3 &axis, const idVec3 &origin, float modelM
     modelMatrix[15] = 1;
 }
 
-// FIXME: these assume no skewing or scaling transforms
+// FIXME: these assume no skewing or scaling transforms (disabled SSE intrinsics on MAC ppc)
 void R_LocalPointToGlobal( const float modelMatrix[16], const idVec3 &in, idVec3 &out ) {
-#if !( defined(MACOS_X) || defined(__APPLE__) )
+#if !( defined(__ppc__) || defined(MACOS_X) || defined(__APPLE__) )
```

Comment:
I don't think this is a very good check for this purpose. It doesn't do what you want on
* x86 Macs (there it should be enabled)
* non-x86 CPUs that aren't PPC (unless they happen to run OSX)

Can't you just use `#ifdef __SSE__`?

Reply:
yeah i newer coded for MAC before so was a bit unsure how to proceed :) ill see if just using __SSE__ helps.